### ISSUE TITLE
docker-compose

### ISSUE THREAD

**Issue by cl-consulting** [spantaleev/matrix-docker-ansible-deploy#1593]
Hello,
This project looks very good. I don’t know Ansible, we use docker-compose on ours infrastructures. We would like to try Matrix & Jitsi, to begin with, but from a docker-compound in order to be able to isolate it, in case of problem with the existing containers. Is this possible? We already have used containers: Nginx, Postgres, etc...
Thank you and congratulations again for this wonderful integration work.

**Comment by spantaleev** [comment]
No, we don't use docker-compose in this project. You can search our issues for a hacky workaround (generating `docker-compose.yml` from systemd `.service` files), but I doubt that will work well.
